doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -273,8 +273,7 @@ Creating Hints
      :cmd:`Import` or :cmd:`Require` the current module.
 
    + :attr:`export` hints are visible from other modules when they :cmd:`Import` the current
-     module, but not when they only :cmd:`Require` it. This attribute is supported by
-     all `Hint` commands except for :cmd:`Hint Rewrite`.
+     module, but not when they only :cmd:`Require` it.
 
    + :attr:`global` hints are visible from other modules when they :cmd:`Import` or
      :cmd:`Require` the current module.
```

Comment:
Maybe it could be worth clarifying in the manual that is support is new. This could be done like this:

```suggestion
     :cmd:`Require` the current module.

   .. versionadded:: 8.14

      The :cmd:`Hint Rewrite` now supports locality attributes like other `Hint` commands.
```

Also, the deprecation note below could be updated to clarify that `Hint Rewrite` without a locality are only deprecated since 8.14.